testfixtures.go

```diff
@@ -110,17 +111,44 @@ func Database(db *sql.DB) func(*Loader) error {
 	}
 }
 
+type DialectOptions func(h helper) error
+
+func WithPlaceHolder(placeholder string) DialectOptions {
```

Comment:
```suggestion
func WithPlaceholder(placeholder string) DialectOptions {
```

Reply:
Add some documentation to this function

Reply:
done

Reply:
@peakle please also address two other comments